u8timeseries/timeseries.py

```diff
@@ -312,10 +301,9 @@ def rescale_with_value(self, value_at_first_step: float) -> 'TimeSeries':
         Numerical imprecisions appear with [value_at_first_step] > 1e+24
 
         :param value_at_first_step: The new value for the first entry of the TimeSeries.
-        :return: A new TimeSeries, whose first value was changed to [value_at_first_step] and whose others values
-                have been scaled accordingly.
+        :return: A new TimeSeries, where the first value is [value_at_first_step] and other values
```

Comment:
The brackets should be changed with quotation marks (so their appear in italic in the documentation :) )

Reply:
Good one, thanks. I plan to anyway rework all the comments in the codebase to follow Numpy style.